### PR TITLE
Add recorder support

### PR DESCRIPTION
**Pull request by muqingCai**
## Summary by CodeRabbit

- **New Features**
	- Enhanced request processing in RPC operations with start and stop recording features for better insight.
- **Updates**
	- Updated dependency versions to improve stability and performance.
- **Bug Fixes**
	- Ensured proper initialization and cleanup of recording contexts in RPC operations.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -22,7 +22,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.alipay.sofa</groupId>
     <artifactId>bolt</artifactId>
-    <version>1.6.9</version>
+    <version>1.6.10-SNAPSHOT</version>
```
Comment (chuailiwu) [inline pom.xml:25]: 验证OK了直接用1.6.10吧

File: pom.xml [hunk 2]
```diff
@@ -85,7 +85,7 @@
 
         <project.encoding>UTF-8</project.encoding>
         <slf4j.version>1.7.21</slf4j.version>
-        <sofa.common.tools>1.0.12</sofa.common.tools>
+        <sofa.common.tools>1.4.0</sofa.common.tools>
```
Comment (OrezzerO) [inline pom.xml:88]: this version does not exist in https://mvnrepository.com/artifact/com.alipay.sofa.common/sofa-common-tools

**Review by OrezzerO — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (muqingCai) [reply pom.xml:88]: sofa-common-tools 1.4.0 has been released

File: pom.xml (hunk 1 above)
Comment (muqingCai) [reply pom.xml]: done

**Review by chuailiwu — APPROVED** [review]
lgtm

**Pull request merged**